internal/analysis/rundynamic.go

```diff
@@ -0,0 +1,111 @@
+package analysis
+
+import (
+	"github.com/ossf/package-analysis/internal/dynamicanalysis"
+	"github.com/ossf/package-analysis/internal/log"
+	"github.com/ossf/package-analysis/internal/pkgecosystem"
+	"github.com/ossf/package-analysis/internal/sandbox"
+)
+
+type DynamicAnalysisResults map[pkgecosystem.RunPhase]*dynamicanalysis.Result
+
+// RunDynamicAnalysis
```

Comment:
Go comments have an official style: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

e.g:

```
// RunDynamicAnalysis runs dynamic analysis for the supplied package pkg in the sandbox sb.
//
// On success it returns ...
```

Private methods (starting with lower-case) do not require commentary, but it can be included.

Reply:
Will fix the comments format